retis/src/collect/collector/ovs/bpf/include/ovs_operation.h

```diff
@@ -123,7 +123,6 @@ static __always_inline struct user_upcall_info *batch_current(struct upcall_batc
 	/* In some cases the branch get optimized. The barrier below
 	 * appear to prevent that making the verifier happy.
 	 */
```

Comment:
Can you remove the comment as well?

Reply:
yeah, removed it, thanks. I was too focused on "whether it works with the older distros" side  :)